src/main/java/seedu/duke/gui/Launcher.java

```diff
@@ -0,0 +1,9 @@
+package seedu.duke.gui;
+
+import javafx.application.Application;
+
+public class Launcher {
```

Comment:
Hi, I really like how clean your code is! Perhaps you could add Javadocs for this class to explain its purpose?